common/src/main/java/com/redhat/cloud/notifications/openbridge/BridgeHelper.java

```diff
@@ -75,7 +80,13 @@ public BridgeAuth getAuthToken() {
             return new BridgeAuth("- OB not enabled token -");
         }
 
-        BridgeAuth ba = new BridgeAuth(getAuthTokenInternal());
+        BridgeAuth ba = null;
+        try {
+            ba = new BridgeAuth(getAuthTokenInternal());
+        } catch (Exception e) {
+            LOGGER.warn("Failed to get an auth token: " + e.getMessage());
```

Comment:
```suggestion
            LOGGER.warn("Failed to get an auth token", e);
```

Reply:
I don't think the full trace is needed, message should be enough

Reply:
Same answer as [here](https://github.com/RedHatInsights/notifications-backend/pull/1100#discussion_r836177503).